sky/core.py

```diff
@@ -261,7 +261,7 @@ def queue(cluster_name: str,
     handle = _check_cluster_available(cluster_name, 'getting the job queue')
     backend = backend_utils.get_backend_from_handle(handle)
 
-    logger.info(f'\nSky Job Queue of Cluster {cluster_name}')
+    print(f'\nJob queue of cluster {cluster_name}')
```

Comment:
Do we want to change the info to `Fetching job queue of cluster {cluster_name}`, so that the user will see a more accurate output when using the API?

Reply:
In the CLI view
```
Job queue of cluster c2
ID  NAME     SUBMITTED    STARTED      DURATION          RESOURCES     STATUS     LOG
20  sky-cmd  18 hrs ago   18 hrs ago   < 1s              1x [CPU:1]    SUCCEEDED  ~/sky_logs/sky-2022-08-10-17-16-20-247074
19  -        19 hrs ago   19 hrs ago   < 1s              1x [CPU:0.5]  SUCCEEDED  ~/sky_logs/sky-2022-08-10-16-32-09-948710
```

I kind of think of this message as the table header, rather than "fetching...".  The table header looks nicer to me. Wdyt? 

In API:
```
Job queue of cluster c2
[{'job_id': 20, 'job_name': 'sky-cmd', 'username': 'zongheng', 'submitted_at': 1660176983.1848047, 'status': <JobStatus.SUCCEEDED: 'SUCCEEDED'>, 'start_at': 1660176987.942061, 'end_at': 1660176988.660497, 'resources': '1x [CPU:1]',...
```
it seems acceptable too, esp. since so far it's not widely used. Wdyt?

Reply:
It makes sense for the CLI case, but for API, since the return value will not be printed, it may be a little bit weird to  only show `Job queue of cluster c2`, without the return value, but it make sense to keep it as before, as not many user using it yet.

Reply:
Moved logging out of core.py into cli.py. This pattern makes more sense and we can adopt it incrementally.